exo-go/src/runtools/run_and_log.go

```diff
@@ -0,0 +1,25 @@
+package runtools
+
+import (
+	"os"
+
+	execplus "github.com/Originate/go-execplus"
+	shellwords "github.com/mattn/go-shellwords"
+)
+
+// AndLog runs the given command, logs the process to the given
+// channel, waits for the process to finish and returns an error (if any)
+func AndLog(dir string, env []string, logChannel chan string, commandWords ...string) error {
```

Comment:
Thoughts on keeping the name `runtools.RunAndLog` since there is `runtools.Run`?